src/nunavut/cli.py

```diff
@@ -179,13 +185,23 @@ def _build_post_processor_list_from_args(args: argparse.Namespace) \
                     sys.stdout.write(';')
         return 0
 
-    if _should_generate_support(args):
+    if _should_generate_support(args, language_context):
         support_generator.generate_all(is_dryrun=args.dry_run,
                                        allow_overwrite=not args.no_overwrite)
 
     if args.generate_support != 'only':
         generator.generate_all(is_dryrun=args.dry_run,
                                allow_overwrite=not args.no_overwrite)
+
+    # If docgen selected, warn about linked namespaces
+    if args.target_language == 'docgen':
+        if len(extra_includes) > 0:
+            logging.warn(textwrap.dedent("""
+                W: Other lookup namespaces are linked in these generated docs.
+                If you do not generate docs for these other namespaces as well, links to external
+                data types could be broken (expansion will still work).
+            """).lstrip())
```

Comment:
```suggestion
            logging.warning(
                "Other lookup namespaces are linked in these generated docs. "
                "If you do not generate docs for these other namespaces as well, "
                "links to external data types could be broken (expansion will still work)."
            )
```

This minor change fixes unidiomatic formatting, removes the unnecessary `W: ` prefix, and replaces the deprecated `warn` with `warning`.